libr/anal/fcn.c

```diff
@@ -228,7 +228,7 @@ static bool is_delta_pointer_table(RAnal *anal, RAnalFunction *fcn, ut64 addr, u
 		if (o_reg_dst && reg_src && o_reg_dst->offset == reg_src->offset && omov_aop.disp != UT64_MAX) {
 			// Special case for indirection
 			// lea reg1, [base_addr]
-			// mov reg2, (sz)word [reg1 + tbl_off*sz + casetbl_loc_off]
+			// movzx reg2, byte [reg1 + tbl_off + casetbl_loc_off]
```

Comment:
Why this change? Since you add a *new* type of indirect jump table. Why change the old comment?

Reply:
I noticed the comment was wrong, so I fixed it